quick_start_new.md

```diff
@@ -0,0 +1,29 @@
+## Setup
+Compatible with Python versions 3.8.10 and higher
+### Install dependencies
+```bash
+# Create an virtual environment 
+# (optional but recommended)
+$ python -m venv venv 
+$ source venv/bin/activate
+# install dependencies
+(venv) $ pip install -r requirements_quick_start.txt
+```
+
+## Build Data Profiles
+Config data sources in ver/.dsessions/sources/data_config.yml
+
+```bash
+python ver_cli.py profile data_config output_profiles_json/json
```

Comment:
The additional `/json` on `output_profiles_json` is unnecessary, you can try using the new `store_type` 3 instead of 1.

Reply:
Nice catch!